src/containers/namespace-detail/namespace-detail.tsx

```
@@ -340,11 +340,19 @@ export class NamespaceDetail extends React.Component<IProps, IState> {
         this.context.selectedRepo,
       ),
       NamespaceAPI.get(this.props.match.params['namespace']),
-      MyNamespaceAPI.get(this.props.match.params['namespace']).catch(
+      MyNamespaceAPI.get(this.props.match.params['namespace']).catch((e) => {
+        // TODO this needs fixing on backend to return nothing in these cases with 200 status
+        // if view only mode is enabled disregard errors and hope
+        if (
+          this.context.settings.GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_ACCESS
```

this should check that you are actually using the anonymous user, otherwise it eats all errors when the setting is enabled

---

Fixed :)